src/utest/mod.rs

```diff
@@ -7,47 +7,35 @@ use crate::println;
 use core::panic::PanicInfo;
 use qemu_exit::QEMUExit;
 
-static UTEST_SUCESS: &str = "\x1b[32mOK\x1b[0m";
-static UTEST_FAILURE: &str = "\x1b[31mKO\x1b[0m";
+static UTEST_SUCESS: &str = "\x1b[32mok\x1b[0m";
+static UTEST_FAILURE: &str = "\x1b[31mFAILED\x1b[0m";
 
-/// Assert the equality of two elements
-macro_rules! kassert_eq {
-    ($l_exp: expr, $r_exp: expr) => {{
-        crate::utest::uassert_eq($l_exp, $r_exp, "anonymous test")
-    }};
-    ($l_exp: expr, $r_exp: expr, $name: tt) => {{
-        crate::utest::uassert_eq($l_exp, $r_exp, $name)
-    }};
+pub trait Testable {
+    fn run(&self) -> ();
```

Comment:
```suggestion
    fn run(&self);
```

Any reason why this doesn't work or is different ?